### ISSUE TITLE
QUESTION: Can I use a sync chain on Brave Nightly

### ISSUE THREAD

**Issue by privatedev11** [brave/brave-browser#16714]
I want to switch to Brave Nightly from Brave Release but I will only switch if I can do a sync chain and transfer bookmarks.

**Comment by tortino** [comment]
I'm currently using nightly, and have enabled sync.
It seems to works, but maybe some developers could confirm that?

**Comment by privatedev11** [comment]
> I'm currently using nightly, and have enabled sync.
> It seems to works, but maybe some developers could confirm that?
> 

Ill give it a go

**Comment by privatedev11** [comment]
Works on android 11

**Comment by darkdh** [comment]
Yes, you can join the same sync chain between different channels.

**Comment by privatedev11** [comment]
Brill

**Comment by privatedev11** [comment]
Thanks
